examples/plotting/magnetogram_active_regions.py

```diff
@@ -48,7 +48,12 @@
 
 fig = plt.figure()
 ax = fig.add_subplot(projection=smap)
-smap.plot(axes=ax, vmin=-120, vmax=120)
+# Passing vmin/vmax to ``plot`` does not work since
+# a normalisation is set on the map. So we have to
+# work around it like so:
+smap.plot_settings["norm"].vmin=-150
+smap.plot_settings["norm"].vmax=150
```

Comment:
What happens if you leave it alone and just let the map source set the symmetric vmin/vmax?

Reply:
I did that, it sets it to be +-2k and looks very washed out